lib/MusicBrainz/Server/Entity/URL/ASIN.pm

```diff
@@ -27,14 +27,16 @@ sub sidebar_name { shift->pretty_name }
 override affiliate_url => sub {
     my $self = shift;
 
-    my $url = super()->clone;
+    my $url = super();
```

Comment:
In this subclass, it wasn’t done wrong; but it’s slightly more efficient to only clone when actually changing the URL.